molecule/shared/substrate/lxd/create.yml

```diff
@@ -0,0 +1,84 @@
+---
+- name: Manage LXD containers
+  hosts: localhost
+  gather_facts: false
+  connection: local
+  vars:
+    molecule_inventory:
+      molecule:
+        hosts: {}
+    molecule_yml:
+      platforms:
+        - name: instance
+          image: ubuntu:20.04
+          user: ansible
+
+  tasks:
+    - name: Ensure LXD is installed
+      ansible.builtin.command:
+        cmd: snap install lxd
+      become: true
+
+    - name: Initialize LXD (if not already initialized)
+      command: lxd init --auto
+      args:
+        creates: /var/lib/lxd/lxd.db
+      become: true
```

Comment:
In the other molecule create.ymls, I tried to not do any of the installation steps here because depending on the developers work environment it could fail. For example, this would fail if they don't have snap installed. And would continue until they got it installed which might not be possible. They may also prefer a different package manager all together (homebrew, nix, etc)

This file should simply try to use lxd to create a VM and then update (dump) the molecule/shared/inventory/molecule_inventory. If LXD is not installed should simply fail with errors.

Reply:
updated lxd virtualization